config/samples/config.yaml

```diff
@@ -18,6 +18,8 @@ leader_election:
 
 metrics_addr: ":8080"                   # The address the metrics endpoint binds to.
                                         # The default value is ":8080".
+enable_server: false                    # The debug API is behind this server which is disabled by default for security reasons.
+server_addr: ":9092"
```

Comment:
127.0.0.1?
Should we emphasize the purpose of debugging on the fields?

Reply:
The purpose is generic. Although currently it's to debug the in memory state of adc config. Maybe we can add a list
1. `/debug` can be used to debug in-memory state of translated adc configs to be synced with data plane.
....later if handlers are added, we can extend the list